README.md

```diff
@@ -1,16 +1,20 @@
-openstack.net
-=============
-### An OpenStack SDK for Microsoft .NET
+# openstack.net: An OpenStack SDK for Microsoft .NET
 
-<p>openstack.net is an SDK written for the Microsoft .NET platform, designed to enable developers to seamlessly work with the many services provided by the openstack cloud operating system.</p>  
-<p>The openstack.net SDK contains:
-	<ul>
-		<li>A language API</li>
-		<li>Getting Started Guide</li>
-		<li>API Reference Manual</li>
-		<li>Release Notes</li>
-		<li>Sample code.</li>
-	</ul>
-</p>
+openstack.net is an SDK written for the Microsoft .NET platform, designed to enable developers to seamlessly work with
```

Comment:
I'd love to see the sentance start with a capital so maybe:  
The SDK openstack.net, written for for the Microsoft .NET platform, is designed to enable ...